clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -683,21 +686,55 @@ public Node provide() {
         }
     }
 
+    class RetryContext {
```

Comment:
nit: Can we change `RetryContext` to be `CallRetryContext` so it's more explicit as to what we are referring to for the retry context?

Reply:
Make sense. Refactored